### DISCUSSION TITLE
Проблемы и предложения по улучшению программы

### DISCUSSION THREAD

**Discussion by tjden88** [tjden88/JoyMapper#6] [Announcements]
Оставляйте здесь свои отзывы, глюки и предложения по улучшению программы

**Comment by SankohTew** [comment]
Программа не работает. Всё настроил, всё запустил, тестирую в блокноте, браузере, игре - ничего.

**Reply by tjden88** [reply]
Слишком мало информации. Пожалуйста, отправьте мне файл Config.json из директории программы для изучения проблемы

**Comment by Someone-G** [comment]
В ОС Windows 8.1 некорректно отображаются кнопки управления окном программы: свернуть, увеличить, закрыть.
![2024-11-30_190220](https://github.com/user-attachments/assets/01ebae15-30b6-4370-93c7-390219e2fd42)
